### PR TITLE
Update DDP dynamo debug docs

### PR DESCRIPTION
**Pull request by oraluben**
Update `TORCH_LOGS` information for ddp

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

### CODE REVIEW

**Comment by bdhirsh** [comment]
tagging @wconstab for compiler <> distributed logging

File: docs/source/notes/ddp.rst [hunk 1]
```diff
@@ -221,8 +221,8 @@ to fire in-between sections of backwards, and schedule communications to overlap
 
 See `this blog post <https://dev-discuss.pytorch.org/t/torchdynamo-update-9-making-ddp-work-with-torchdynamo/860/1>`_ for
 a more in-depth explanation and experimental results, or read the docs and code at
-`torch/_dynamo/optimizations/distributed.py <https://github.com/pytorch/pytorch/blob/4908a12542798a3e8641faae6b74f068fdfc6778/torch/_dynamo/optimizations/distributed.py#L56>`_
+`torch/_dynamo/optimizations/distributed.py <https://github.com/pytorch/pytorch/blob/bbc39b7bb48d28d67e3253a89cc82df3687ddd1b/torch/_dynamo/backends/distributed.py#L124>`_
 
-To Debug DDPOptimizer, set `torch._dynamo.config.log_level` to DEBUG (for full graph dumps) or INFO
+To Debug DDPOptimizer, set `TORCH_LOGS='+dynamo'` (for full graph dumps) or `TORCH_LOGS='dynamo'`
```
Comment (wconstab) [inline docs/source/notes/ddp.rst:226]: actually there is a `ddp_graphs` log field which is probably the one you want to talk about here.  I'm not sure if adding `dynamo` to TORCH_LOGS does anything to the DDPOptimizer?

Comment (oraluben) [reply docs/source/notes/ddp.rst]: Thanks! You're right.

The `ddp_graphs` seems to (only) print the graph, the bucket size is printed via `dynamo`:
https://github.com/pytorch/pytorch/blob/d5d62e85615fdf345e0556a9d8edbee2d3c64ae2/torch/_dynamo/backends/distributed.py#L13

**Comment by oraluben** [comment]
I suggest to use one single logger for the bucket and ddp graph, with `TORCH_LOGS='ddp'` and `TORCH_LOGS='+ddp'` to enable each. Does that look better to you? @wconstab

**Comment by wconstab** [comment]
let me bump the discussion on this issue https://github.com/pytorch/pytorch/issues/113544 and get a conclusion on whether we want to pile everything into 'distributed' topic or separate them out by components.  then you can make the update based on that.

(or if you don't want to wait i'll also stamp this PR for changing to put both `ddp_graphs and dynamo` in the docs so its correct for the time being)

**Comment by wconstab** [comment]
after playing with it a bit, i think it is appropriate for `ddp_graphs` to  remain separate because it is an "artifact logger".

however, i am making a change to include ddpoptimizer's regular logger (separate from graphs) in  the 'distributed' component. Will PR shortly and link here.

**Comment by oraluben** [comment]
> however, i am making a change to include ddpoptimizer's regular logger (separate from graphs) in the 'distributed' component. 

Sounds good, feel free to include the changes here, I believe you can integrate all the code and doc changes more effectively. I can also update this PR after yours being merged.

**Comment by wconstab** [comment]
oops i think i dropped this.  @oraluben any interest in rebasing this and merging? or want me to copy the changes over to a new PR and land it

**Comment by oraluben** [comment]
@wconstab Sure, you can just copy the changes to your pr :)

**Comment by wconstab** [comment]
replaced by https://github.com/pytorch/pytorch/pull/118295.  (would have updated this PR and merged but i dont have permissions to your fork)

**Pull request closed without merging**